README.rst

```diff
@@ -159,7 +159,9 @@ Exclude from search and elevation
 ---------------------------------
 
 By default this add-on introduces two new fields to the default content types
-or any custom type derived from ATContentTypes.
+or any custom type derived from ATContentTypes. For dexterity content types,
+including plone.app.contenttypes, a behavior is provided which must be manually
+activated on the desired content types.
```

Comment:
It's a regular behavior, so I'm pretty sure you can do both. My understanding was that the usual way to activate a behavior is on a content type.